src/dapla_metadata/variable_definitions/utils/descriptions.py

```diff
@@ -0,0 +1,70 @@
+"""Utilities for dynamically adding extra fields to Pydantic models, specifically Norwegian descriptions."""
+from pathlib import Path
+from typing import cast
+
+import yaml
+from pydantic import Field
+from pydantic.config import JsonDict
+
+from dapla_metadata.variable_definitions.utils.constants import (
+    VARDEF_DESCRIPTIONS_FILE_PATH,
+)
+from dapla_metadata.variable_definitions.variable_definition import CompletePatchOutput
+from dapla_metadata.variable_definitions.variable_definition import VariableDefinition
+
+
+def load_descriptions(file_path_str: str) -> dict:
+    """Load and return the contents of a YAML file as a dictionary.
+
+    Args:
+        file_path_str (str): Path to the YAML file.
+
+    Returns:
+    dict: Parsed contents of the YAML file.
+    """
+    file_path = Path(file_path_str)
+    with Path.open(file_path, encoding="utf-8") as f:
+        return yaml.safe_load(f)
+
+
+# Loads when module is imported
+DESCRIPTIONS = load_descriptions(VARDEF_DESCRIPTIONS_FILE_PATH)
+
+
+def apply_norwegian_descriptions_to_model(
+    model: type[CompletePatchOutput] | type[VariableDefinition],
+) -> None:
+    """Add Norwegian descriptions to the fields of a Pydantic model.
+
```

Comment:
It appears that this function **globally** modifies the models in place?

I think that's alright, but it would be good to have a note of that in the docstring since it can potentially lead to surprising behaviour elsewhere the models are used.